config/java.header

```diff
@@ -12,7 +12,7 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 // Lesser General Public License for more details.
 //
-// You should have received a copy of the GNU Lesser General Public
-// License along with this library; if not, write to the Free Software
+// You will recieve a copy of the GNU Lesser General Public
```

Comment:
https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
`You should have received a copy of the GNU Lesser General Public
License along with this library; `